stable/goldilocks/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
-appVersion: "v4.0.0"
-version: 4.0.0
+appVersion: "v4.0.1"
```

Comment:
This shouldn't change, we haven't released 4.0.1 of goldilocks yet.

Reply:
Done